modules/update-service-overview.adoc

```diff
@@ -61,6 +61,7 @@ determine if updates are available.
 ====
 Reverting your cluster to a previous version, or a rollback, is not supported.
 Only upgrading to a newer version is supported.
+If your upgrade fails, contact Red Hat support.
```

Comment:
I do not think they should call for each upgrade failure , but may be for the ones where availability goes for a toss. Because if it is minor issue they can wait for an available update which might fix the issue.

Reply:
this line is copy/pasted from [existing content][1], which is still current.  I agree that we don't formally define "fails" to be "ClusterVersion Failing=True", but...  That's a page-at-midnight `ClusterOperatorDown{name="version"}` error.  I think "I got paged at midnight" is enough that opening a case with RH isn't overreacting ;).

[1]: https://github.com/openshift/openshift-docs/pull/13993/files#diff-c53e73b13c36b252211983803b66aa8af998646d08f67ef3ff2f6ae86d1c84faR138